src/components/stacking.tsx

```diff
@@ -1,88 +1,88 @@
 import * as React from 'react';
 import { Box, Flex, color, Stack, StxInline, Circle } from '@stacks/ui';
 import { Caption, Text, Link, Title } from '@components/typography';
-import { useStacksInfo } from '@common/hooks/use-stacks-info';
+// import { useStacksInfo } from '@common/hooks/use-stacks-info';
```

Comment:
Did you mean to leave this file commented out?

Reply:
ah thank you! i will finish that up